Services/Concrete/ThirdParties/CsgoDashStatsComService.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Threading.Tasks;
+using Core.Models;
+using Services.Interfaces;
+using Services.Models.ThirdParties;
+
+namespace Services.Concrete.ThirdParties
+{
+    class CsgoDashStatsComService : IThirdPartyInterface
+    {
+        private const string ENDPOINT = "https://csgo-stats.net/api/match/status";
```

Comment:
This variable is not used and so should be removed.

Reply:
Removed 👍